client/src/components/mapping/InputColumn/index.tsx

```diff
@@ -215,31 +213,33 @@ const InputColumn = ({ input, schema, source }: Props) => {
                   }}
                 />
               </div>
-              <div className="stacked-tags">
-                <Tag>CONCEPT MAP</Tag>
-                <ButtonGroup>
-                  <Button
-                    text={input.conceptMap ? input.conceptMap.title : 'None'}
-                    onClick={(_e: React.MouseEvent) => {
-                      setConceptMapOverlayVisible(true);
-                    }}
-                  />
-                  <Button
-                    className="delete-button"
-                    icon="cross"
-                    minimal={true}
-                    disabled={!input.conceptMapId}
-                    onClick={(_e: React.MouseEvent) => {
-                      updateInput({
-                        variables: {
-                          inputId: input.id,
-                          data: { conceptMapId: null }
-                        }
-                      });
-                    }}
-                  />
-                </ButtonGroup>
-              </div>
+              {attribute.types[0] === 'code' ? (
```

Comment:
I think there you can use the `&&` syntax : `{attribute.types[0] === 'code' && (...)}`